src/react/src/components/Contribute/ProductionLocationInfo.jsx

```diff
@@ -784,6 +815,7 @@ const ProductionLocationInfo = ({
                                 handleProductionLocation(inputData, osID);
                             }}
                             className={classes.submitButtonStyles}
+                            disabled={!isFormValid}
```

Comment:
As I understand the design in Figma, it should work as follows:

1. Initially, the Submit button is enabled.
2. The user clicks the button, but the required fields are not filled.
3. An error message displays. See the screenshot ->
<img width="937" alt="Screenshot 2025-02-21 at 12 37 35" src="https://github.com/user-attachments/assets/2145029e-5090-4ff4-bc2f-ae4d7cc548d5" />


Otherwise, if the submit button is disabled, it may not be clear what the user needs to do next. Less experienced internet users might not understand what to do. Only experienced users will likely notice the asterisks and understand that those are the required fields. @protsack-stephan @mazursasha1990 what do you think? Do you think it is not so important fow now?